style.css

```diff
@@ -406,3 +406,123 @@ body {
   margin-left: 15px;
   margin-right: 30px;
 }
+
+.contact-form {
+  width: 90%;
+  height: auto;
+  padding-left: 15px;
+  padding-right: 15px;
+  background-color: #6070ff;
+  background-image: url(/images/contact\ form-mobile.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-top-left-radius: 80px;
+}
+
+.contact-me {
+  display: flex;
```

Comment:
<img width="439" alt="1" src="https://github.com/ariful04/Myportfolio/assets/77843050/a2010fa8-d5cd-49d1-8154-9083b9554671">

- [ ] You have a blank space on the right of your contact form, please remove it as it is not in the original design

Reply:
Hello @joskalenda thank you for your nice feedback, I update my html file in form URL section and css file width problem as your requirement, But image path is correct my side and my collaborator side please check again.